code/game/machinery/iv_drip.dm

```diff
@@ -237,7 +237,7 @@
 
 	. += span_notice("[attached ? attached : "No one"] is attached.")
 
-/obj/machinery/iv_drip/screwdriver_act(mob/living/user, obj/item/I)
+/obj/machinery/iv_drip/attacked_by(obj/item/attacking_item, mob/living/user)
```

Comment:
use attackby and check if the attacking item is specifically the mask being used